docker-compose.yml

```diff
@@ -4,11 +4,12 @@ x-database-image: &database-image
 x-file-storage-image: &file-storage-image
   image: rclone/rclone:${RCLONE_VERSION:-1.68}
 
+
 services:
   app:
     image: dhis2/core:${DHIS2_VERSION:-42}
     restart: unless-stopped
-    user: 65534:65534
+    user: nobody:nobody
```

Comment:
I'm pretty sure this is the reason for the pipeline failing. The distroless image doesn't have a /etc/passwd file (I'm pretty sure, but if it does the user nobody isn't listed there). The solution is to stick with the id's here